requirements-test.txt

```diff
@@ -1,11 +1,12 @@
+--requirement requirements.txt
 black
 coverage
-Faker
```

Comment:
This seems like a test thing that should be here instead of in `requirements.txt`.